app/components/form_components/autocomplete/script.js

```diff
@@ -1,21 +1,31 @@
 import './style.scss'
 import accessibleAutocomplete from 'accessible-autocomplete'
 import { nodeListForEach } from 'govuk-frontend/govuk/common'
+import sort from './sort.js'
 
 const $allAutocompleteElements = document.querySelectorAll('[data-module="app-autocomplete"]')
 const defaultValueOption = component => component.getAttribute('data-default-value') || ''
 
-const suggestionTemplate = (value) => {
-  const matches = /(.+)\(([^)]+)\)/.exec(value)
-  if (matches) {
-    return `<span>${matches[1]}</span> <strong>(${matches[2]})</strong>`
-  } else {
-    return `<span>${value}</span>`
+const suggestion = (value, options) => {
+  const option = options.find(o => o.name === value)
+  if (option) {
+    return option.append ? `<span>${value}</span> ${option.append}` : `<span>${value}</span>`
+  }
+}
+
+const enhanceOption = (option) => {
+  return {
+    name: option.label,
+    synonyms: (option.getAttribute('data-synonyms') ? option.getAttribute('data-synonyms').split('|') : []),
```

Comment:
Should probably have a test for multiple synonyms here, otherwise the `.split('|')` is a bit mysterious

Reply:
writing one right now! i think it will also enlighten the comment above.

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/pull/949#discussion_r647565954

Reply:
Added!